created new jaas onboarding page

matthewhelmke opened the pull request:
@juju/docs
ready for review

evilnick commented on src/en/getting-started-jaas.md:
```diff
@@ -0,0 +1,127 @@
+
+# Getting started with Juju as a Service
+
+Juju as a Service (JAAS) is the fastest and easiest way to quickly get started
```
fastest ANd quickly?

I would just say fastest and easiest way to model and deploy your cloud-based applications

matthewhelmke replied:
I like that!

evilnick commented on src/en/getting-started-jaas.md:
```diff
@@ -0,0 +1,127 @@
+modeling and deploying your cloud-based applications. When you use JAAS,
+Canonical manages the Juju infrastructure. This frees you to concentrate on
+your software and solutions. With JAAS you can deploy, configure, and operate
+your applications on AWS, GCE, and Azure.
```
public clouds like ... 
and I would write out the full names, and probably add links

matthewhelmke replied:
Good call.

evilnick commented on src/en/getting-started-jaas.md:
```diff
@@ -0,0 +1,127 @@
+In order for JAAS to deploy your workload on the cloud of your choice, you must
+enter credentials for each cloud you would like to use.
+
+We recommend that you use each public cloud's identity and access management
```
I think we should just add a list of the clouds and link to the relevant credential page (we will have to review those also, and maybe make alternatives). So scrap this paragraph and do a list for the supported clouds

matthewhelmke replied:
👍

evilnick commented on src/en/getting-started-jaas.md:
```diff
@@ -0,0 +1,127 @@
+(IAM) tool to generate a new set of credentials exclusively for use with JAAS.
+See [Cloud credentials][credentials] for more information. 
+
+## Use JAAS from the web GUI
```
meh. Just ' log in to JAAS'

matthewhelmke replied:
Are you sure, because this parallels nicely with "Use JAAS from the CLI" ?

matthewhelmke replied:
Never mind, I found the request to move the CLI stuff to another page, making my parallel comment obsolete. I can change this.

evilnick commented on src/en/getting-started-jaas.md:
```diff
@@ -0,0 +1,127 @@
+
+## Use JAAS from the web GUI
+
+It is easiest to do the initial steps using the web. Open the [JAAS login page][jaaslogin]
```
scrub the first sentence

matthewhelmke replied:
👍

evilnick commented on src/en/getting-started-jaas.md:
```diff
@@ -0,0 +1,127 @@
+your web browser to log you in. If the tab does not open, check and disable any
+pop-up blockers that may be running and try again.
+
+Press the green + button on the page to design and deploy your model. Press
```
could probably do with some more steps and explanation - this may well be first time using Juju.

matthewhelmke replied:
☑️

evilnick commented on src/en/getting-started-jaas.md:
```diff
@@ -0,0 +1,127 @@
+![juju_jaas_main](./media/juju_jaas_main.png)
+
+The last step in the deployment requires and will request your credentials for
+the public cloud you want to use, so prepare these beforehand and have them
```
again, an example with screenshot would be handy

matthewhelmke replied:
☑️

evilnick commented on src/en/getting-started-jaas.md:
```diff
@@ -0,0 +1,127 @@
+
+View the new model in the JAAS web UI by logging in to [https://jujucharms.com][https://jujucharms.com].
+
+## Collaborate
```
this jumps around too much. Assuming that the cli stuff above moves to a new page, we can simply point at the existing page for sharing models.

evilnick commented on src/en/getting-started-jaas.md:
```diff
@@ -0,0 +1,127 @@
+the public cloud you want to use, so prepare these beforehand and have them
+ready.
+
+## Use JAAS from the CLI
```
I would put this on a new page - next steps:

 Install the Juju cli for more  contol >

we will need to change the getting-started-general page also. Make it just an install page

matthewhelmke replied:
Ahh, I see. In that case, the title change above makes more sense...

matthewhelmke commented:
I have added a new CLI page and moved info to it. I polished the message on each page and I added several new screenshots to the steps in the GUI process. It is still short and sweet, but with far less for the naive user to figure out on his own but not so much info as to frustrate the savvy user.

mitechie commented on src/en/getting-started-jaas.md:
````diff
@@ -16,94 +17,54 @@ when registering which is the username that will be used by JAAS.
-```bash
-juju register jimm.jujucharms.com
-```
+Press the green + button on the page to add a charm or bundle to your model.
````
Let's leave out the charm/bundle and just go with "to start building your model".

matthewhelmke replied:
👍

mitechie commented on src/en/getting-started-jaas.md:
```diff
@@ -16,94 +17,54 @@ when registering which is the username that will be used by JAAS.
-you through entering the pertinent credential data for the cloud you specify,
-as in this example which uses the GCE cloud.
+You will be shown a list of available charms and bundles with a description of
+each. Select a charm or bundle to learn more about it and decide whether to
```
do we need this data on here? "with a description of each"? "decide whether to use..." Can we shrink it down to action based and simplify as much as possible?

matthewhelmke replied:
We can. I'm not convinced we should. This is useful data and isn't terribly long.

mitechie commented on src/en/jaas-cli.md:
````diff
@@ -0,0 +1,62 @@
+as in this example which uses the GCE cloud.
+
+```bash
+juju add-credential google
````
this is the same as juju. Maybe just reference the existing juju docs?

matthewhelmke replied:
👍

mitechie commented on src/en/jaas-cli.md:
```diff
@@ -0,0 +1,62 @@
+
+See [Cloud credentials][credentials] for more information.
+
+To add a new model from the CLI, use:
```
This is also the same as juju.

matthewhelmke replied:
👍

mitechie commented on src/en/jaas-cli.md:
````diff
@@ -0,0 +1,62 @@
+To deploy kubernetes-core, use:
+
+```bash
+juju deploy kubernetes-core
````
and more of the same.

matthewhelmke replied:
👍

mitechie commented on src/en/jaas-cli.md:
````diff
@@ -0,0 +1,62 @@
+If you are using Juju 2.2, use:
+	
+```bash
+juju login JAAS
````
does case matter here? I've not tested that.

matthewhelmke replied:
I don't know. This is how it was given to me. I'll look into it further.

matthewhelmke replied:
Figured it out. The answer is that we should leave it as originally written, 'jaas'. I introduced the error during a search/replace updating all the JaaS references to JAAS and accidentally "fixed" this one, too. Reverting to all lower case.

matthewhelmke commented:
@evilnick sounds good. Done.
